arbor/include/arbor/mechanism.hpp

```diff
@@ -49,8 +53,11 @@ class mechanism {
     // Non-global parameters can be set post-instantiation:
     virtual void set_parameter(const std::string& key, const std::vector<fvm_value_type>& values) = 0;
 
+    // Peek into state variable
+    virtual fvm_value_type* field_data(const std::string& var) = 0;
```

Comment:
the `value_type` is typedefed above, but we are using the `fvm_value_type` here. There should be consistency. Maybe we don't need the typedefs?

Reply:
I had a discussion with @halfflat  some time ago from which I concluded that indeed the plan is to get rid of the `typedef`s  where possible.